src/components/InputVerify.js

```diff
@@ -102,8 +100,9 @@ class InputVerify extends Component {
   };
 
   render() {
+    const { onChange, ...props } = this.props;
```

Comment:
For all the destructured comments, the idea of destructuring the `onChange` is to prevent both `onChange` functions running, causing the double event issue we were having.

So by destructuring you are essentially preventing the `onChange` that was passed to the component from being applied to the Input by default when you include the `{...props}`